liquid.py

```diff
@@ -37,7 +37,8 @@ def main():
         commands.nomad_address,
         commands.deploy,
         commands.resources,
-        commands.gc,
+        commands.cleancollectionjobs,
```

Comment:
I think we should keep the `gc` command named as it was, and have it also remove extra collection jobs. The `nomadgc` addition is really useful here through.

Reply:
This may confusion, as gc, which is generic, doesn't garbage collect all dead jobs, only the collections ones. Maybe `collectionsgc`?